tests/Fixtures/e2e/SymfonyFlex/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "require-dev": {
+        "symfony-flex-demo/flex": "dev-master"
```

Comment:
What is the reason for this package? Couldn't we just go with requiring symfony/flex ?

Reply:
There is no reason - just an accident